src/main/java/seedu/duke/data/Profile.java

```diff
@@ -2,9 +2,62 @@
 
 import java.util.ArrayList;
 
+/**
+ * Represents a Profile for the SplitLah application.
+ */
 public class Profile {
-    // To be completed when Warren pushes
     private ArrayList<Session> sessionList;
     private int sessionIdTracker;
-    private int activityCount;
+    private int activityIdTracker;
+
+    /**
+     * Constructor to create a profile object.
+     */
+    public Profile() {
+        this.sessionList = new ArrayList<>();
+        sessionIdTracker = 0;
+        activityIdTracker = 0;
+    }
+
+    /**
+     * Returns the list of session stored in the profile.
+     * @return The ArrayList of sessions.
+     */
+    public ArrayList<Session> getSessionList() {
+        return sessionList;
+    }
+
+    /**
+     * Returns a new session id for session object to be created.
+     * @return The new session id as int.
+     */
+    public int getNewSessionId() {
+        return sessionIdTracker + 1;
```

Comment:
This is slightly problematic, might not be what you are trying to achieve. Do you mean the following?
```suggestion
        return sessionIdTracker += 1;
```

Or do you require the user to call getNewSessionId(), then updateSessionIdTracker()?